openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -710,6 +706,45 @@ Cache-Control: no-store
 }
 ```
 
+# Nonce Endpoint {#nonce-endpoint}
+
+This endpoint allows a Client to acquire a fresh `c_nonce` value without the overhead of a full Credential Request. A Credential Issuer that requires `c_nonce` values to be incorporated into proofs in the Credential Request (see (#credential-request)) MUST offer a Nonce Endpoint.
+
+The `nonce_endpoint` Credential Issuer Metadata parameter, as defined in (#credential-issuer-parameters), contains the URL of the Credential Issuer's Nonce Endpoint.
+
+
+## Nonce Request {#nonce-request}
+
+A request for a nonce is made by sending an HTTP GET request to the URL provided in the `nonce_endpoint` Credential Issuer Metadata parameter.
```

Comment:
is there any benefit of using GET over POST?

Reply:
is there any benefit of using POST over GET?

Reply:
In OpenID4VP having the request_uri with POST in the first place would have made things easier instead of adding the POST method later

Reply:
A lot of IMHO very questionable things led to request_uri with POST, which itself is IMHO a very awkward construct. And other than being in the umbrella of OpenID4VP, I don't see what they have in common. 

Using a GET to get something seemed like the most straightforward thing to do.

Reply:
I would argue most calls in OpenID4VCI are POST calls?

Reply:
> I would argue most calls in OpenID4VCI are POST calls?

Because they send data in the request.

Reply:
I would agree with GET here as well unless we already have a clear idea that we will need to send data (POST)

Reply:
Not a hill I want to die on, was just a question coming to my mind. If others think GET is better, I'm fine to close this conversation

Reply:
Resolving conversation to avoid any deaths on hills.

Reply:
I filed https://github.com/openid/OpenID4VCI/issues/395

No reason to delay this PR, but I would like to be able to use this endpoint without needing to pull in all of OIDC4VCI.